src/graph/backend/dnnl/kernels/sdp_decomp.cpp

```diff
@@ -51,8 +51,9 @@ status_t sdp_decomp_kernel_t<quantized, dt>::compile_impl(
     BACKEND_DNNL_CHECK(set_given_inputs_outputs(subgraph_, inputs, outputs));
 
     // Check if it's supported by decomposition kernel
-    if (!sdp_cfg_.initial_check(subgraph_, inputs))
-        return status::unimplemented;
+    VCONDCHECK(graph, create, check, sdp_decomp,
+            sdp_cfg_.initial_check(subgraph_, inputs), status::unimplemented,
+            "sdp_decomp_kernel_t: initial check failed");
```

Comment:
I think we don't need this as we already do checks in the `initial_check` function.

Reply:
removed. thanks